CONTRIBUTING.md

```diff
@@ -41,35 +31,34 @@ If you wish to contribute code (features or bug fixes), see the [Submitting a pa
-  `git rebase`; this is important to ensure your commit history is clean and
-   linear)
+- Your commit history is clean: changes are atomic and the git message format was respected
+- Rebase your work on top of the base branch (seek help online on how to use `git rebase`; this is important to ensure your commit history is clean and linear)
+
+If the above guidelines are respected, your Pull Request will be reviewed by a maintainer.
+
+If you are asked to update your patch by a reviewer, please do so! Remember: **you are responsible for pushing your patch forward**. If you contributed it, you are probably the one in need of it. You must be prepared to apply changes to it if necessary.
+
+If your Pull Request was accepted and fixes a bug, adds functionality, or makes it significantly easier to use or understand Kuma's website, congratulations! You are now an official contributor to Kuma. Get in touch with us to receive your very own [Contributor T-shirt](#contributor-t-shirt)!
+
+#### Sign Your Work
+
+The sign-off is a simple line at the end of the explanation for a commit. All commits need to be signed. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. The rules are pretty simple, if you can certify the below (from [developercertificate.org](https://developercertificate.org/)):
+
+To signify that you agree to the DCO for a commit, you add a line to the git commit message:
 
-If the above guidelines are respected, your Pull Request will be reviewed by
-a maintainer. And if you are asked to update your patch by a reviewer, please do so!
+```txt
+Signed-off-by: Jane Smith <jane.smith@example.com>
+```
 
-If your Pull Request was accepted and fixes a bug, adds functionality, or
-makes it significantly easier to use or understand Kuma's website, congratulations!
-You are now an official contributor to Kuma. Get in touch with us to receive
-your very own [Contributor T-shirt](#contributor-t-shirt)!
+In most cases, you can add this signoff to your commit automatically with the `-s` flag to `git commit`. You must use your real name and a reachable email address (sorry, no pseudonyms or anonymous contributions).
 
 ### Contributor T-shirt
 
-If your Pull Request to [kumahq/kuma](https://github.com/kumahq/kuma) was
-accepted, and it fixes a bug, adds functionality, or makes it significantly
-easier to use or understand Kuma, congratulations! You are eligible to
-receive the very special Contributor T-shirt! Go ahead and fill out the
-[Contributors Submissions form](https://goo.gl/forms/5w6mxLaE4tz2YM0L2).
+If your Pull Request to [kumahq/kuma](https://github.com/kumahq/kuma) was accepted, and it fixes a bug, adds functionality, or makes it significantly easier to use or understand Kuma, congratulations! You are eligible to receive the very special Contributor T-shirt! Go ahead and fill out the [Contributors Submissions form](https://goo.gl/forms/5w6mxLaE4tz2YM0L2).
```

Comment:
```suggestion
If your pull request was accepted, and it fixes a bug, adds functionality, or makes it significantly easier to use or understand Kuma, congratulations! You are eligible to receive the very special Contributor T-shirt! Go ahead and fill out the [Contributors Submissions form](https://goo.gl/forms/5w6mxLaE4tz2YM0L2).
```

Reply:
Good call. Looking at the linked form, it actually doesn’t so I’ve removed the paragraph altogether. 👍